tests/test_pysyncrosim.py

```
@@ -76,6 +80,39 @@ def test_session_package_functions():
     pkg_subset = mySession.packages()[mySession.packages()["Name"] == "helloworld"]
     assert "2.0.0" not in pkg_subset["Version"].values
     assert "2.0.1" in pkg_subset["Version"].values
+
+def test_session_restore_function():
+
+    # Test with incorrect Library
+    mySession = ps.Session(session_path)
+    with pytest.raises(ValueError, match=f"Library not found: test"):
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)